Sources/XMTP/ConversationV2.swift

```diff
@@ -188,8 +193,8 @@ public struct ConversationV2 {
 		}
 	}
 
-	public var createdAt: Date {
-		Date(timeIntervalSince1970: Double(header.createdNs / 1_000_000) / 1000)
+	public var conversationCreatedAt: Date {
```

Comment:
Do we still need this? Maybe we can move this logic up? Otherwise I'd say lets not rename this here. I think if this is whats being accessed externally it should stay createdAt
```suggestion
	public var createdAt: Date {
```

Reply:
What are you referring to moving up? This method is needed to get the most recent saved conversation. I would rename it as you suggested